### PR TITLE
:sparkles: New feature: Header components

### PR DESCRIPTION
**Pull request by Quaquaro**
Add the first component to App.js Define tests for every child of the
component and write a story.

### CODE REVIEW

File: src/components/TrackedGamesList.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  height: 45px;
+  position: relative;
+  z-index: 2;
+  border-radius: 20px;
+  margin: 60px 0;
+
+  &::before,
+  ::after {
+    content: '';
+    width: 260px;
```
Comment (ttaannjjaa) [inline src/components/TrackedGamesList.js:36]: Did you intentionally change the size here (250 vs 260 and 45 vs 55)?

**Review by ttaannjjaa — APPROVED** [review]
Hi Chris, awesome job done, i felt the hard work behind this in every line i read. There are some really cool features in your code, which were new for me.

File: src/components/TrackedGamesList.js (hunk 1 above)
Comment (Quaquaro) [reply src/components/TrackedGamesList.js:36]: Yes i did.

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+export default function Button() {
+  return <TrackGameButton>TRACK GAME</TrackGameButton>;
+}
+
+const TrackGameButton = styled.button`
+  background-color: ${(props) => props.theme.color.neonBlue};
+  border-radius: 9999em;
```
Comment (FaGru) [inline src/components/Button.js:8]: why not just 100% at the border? 😅

**Review by FaGru — APPROVED** [review]
nice code and awesome styling 👍🏻

**Pull request merged**